Celeste.Mod.mm/Patches/MapData.cs

```diff
@@ -208,6 +208,11 @@ public patch_MapData(AreaKey area)
             return null;
         }
 
+        public static void ParseTags(BinaryPacker.Element child, Backdrop backdrop) {
```

Comment:
I agree with WEGFan that this method should be private, and personally don't see why a mod would ever want to use this. But on the other hand, there should be no harm in keeping this public.